resources/js/components/Elements/Slider.vue

```diff
@@ -18,20 +19,48 @@
             vertical: {
                 type: Boolean,
                 default: false
-            }
+            },
+            interval: {
+                type: Number,
+                default: 5000,
+            },
+            autoplay: {
+                type: Boolean,
+                default: false,
+            },
+            stopOnHover: {
+                type: Boolean,
+                default: true,
+            },
         },
         data: () => {
             return {
                 position: 0,
                 showLeft: false,
                 showRight: false,
-                mounted: false
+                mounted: false,
+                hover: false,
+                pause: ()=>{},
+                resume: ()=>{}
             }
         },
         mounted() {
-            useEventListener(this.slider, 'scroll', this.scroll)
+            this.slider.addEventListener('scroll', this.scroll)
```

Comment:
This one could stay so we don't need the `beforeDestroy()` as VueUse will take care of that automatically.

cc @indykoning